### DISCUSSION TITLE
Any Success on IBM POWER7 install

### DISCUSSION THREAD

**Discussion by Tlturner77** [kth5/archpower#86] [Q&A]
I updated my drives on my POWER7 system and decided to move from AIX. System start to boot and but then loops. If anyone has had success with this on a IBM POWER7 bare metal let me know. just FYI. I am not using vios with and LPAR.

**Comment by kth5** [comment]
I don't have access to any POWER7 hardware. Recently we were able to test on POWER8 using the `powerpc64` flavour. Theoretically it should work just fine but I would right away also mention, drivers may not be available or function correctly.

**Reply by Oichkatzelesfrettschen** [reply]
@Tlturner77 Which model do you have? I have a POWER7... something in the garage and would be willing to test with you. 

Any special boot instructions for BOOT USB or install DVD that you've found so far? 

@kth5 **Regarding testing and submitting logs for wayward architectures like the POWER7:** what are your expectations from us? What data or information will be helpful in you troubleshooting these issues with us remotely?

**Reply by kth5** [reply]
> @kth5 **Regarding testing and submitting logs for wayward architectures like the POWER7:** what are your expectations from us? What data or information will be helpful in you troubleshooting these issues with us remotely?

We tested and fixed things on a S822 I think and I had access remotely via VPN to both the HMC as well as the primary network interface. The HMC allowed me to use custom ISOs to boot from. That way I could see serial output via HMC to see where things get stuck before SSH becomes available.
